templates/zerver/help/include/create-bot-construct-url.md

```diff
@@ -1,6 +1,11 @@
 Next, on your {{ settings_html|safe }}, create a bot for
-{{ integration_display_name }}. Construct the URL for the
-{{ integration_display_name }} bot using the bot API key and stream name:
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
```

Comment:
I think we should explain why you should do this.  E.g. "Next, on your settings_html, create an [incoming webhook bot](/help/bot-types) ..." and then we can put an explanation for what the various bot types are in the `/help/` page; that's also a bit less verbose.

The actual explanation is roughly "since that limits the bot's API key to only be able to send messages", which is important for security.